src/components/submitComment/index.js

```diff
@@ -1,77 +1,116 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import './styles.css';
 
-const SubmitComment = ({
-  onUserCommentChange,
-  onCommentSubmit,
-  userComment,
-  title = 'Add a comment:',
-  type = 'Add',
-  onCommentCancel,
-  comment = {},
-}) => {
-  const { _id = null, userEmail = '', userName = '' } = comment;
+class SubmitCommentForm extends Component {
+  constructor(props) {
+    super(props);
 
-  const formTitle =
-    type === 'Edit' ?
-      <h2>{title}: <span>{`${userName} - ${userEmail}`}</span></h2>
-      :
-      <h2>{title}</h2>
-    ;
+    this.state = {
+      hasErrors: false
+    };
 
-  return (
-    <section className="blabbr-submitComment">
-      <form>
-        {formTitle}
-        <textarea
-          value={userComment}
-          onChange={onUserCommentChange}
-        />
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
 
-        { type === 'Edit' ?
-        [
+  handleChange(event) {
+    this.setState({ hasErrors: false });
+    this.props.handleChange(event.target.value);
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    event.stopPropagation();
+
+    if (!this.refs.userComment.value ||
+      this.refs.userComment.value.length < 1) {
+      this.setState({
+        hasErrors: true
+      });
+    } else {
+      this.props.handleSubmit(event.target.id);
+    }
+  }
+
+  render() {
+    const {
+      userComment,
+      title = 'Add a comment:',
+      type = 'Add',
+      onCommentCancel,
+      comment = {}
+    } = this.props;
+
+    const {
+      _id = null,
+      userEmail = '',
+      userName = ''
+    } = comment;
+
+    const formTitle =
+      type === 'Edit' ?
+        <h2>{title}: <span>{`${userName} - ${userEmail}`}</span></h2>
+        :
+        <h2>{title}</h2>
+      ;
+
+    return (
+      <section className="blabbr-submitComment">
+        <form>
+          {formTitle}
+          <textarea
+            value={userComment}
+            onChange={this.handleChange}
+            ref="userComment"
+          />
+          { this.state.hasErrors && 
+            <div className="error">You need to enter a comment.</div>
+          }
+          { type === 'Edit' ?
+          [
```

Comment:
Maybe put this in some vars and a function or something? Looks a bit dirty right in the JSX.